### ISSUE TITLE
Accessibility and Screen Reader Issues on Embed

### ISSUE THREAD

**Issue by blairlierman** [washingtonstateuniversity/WSUWP-Plugin-iDonate#181]
**Accessibility Issues**
- Placeholders for Start Date, End Date, Additional Comments, and Select your employer do not meet ADA compliance for contrast 

**Screen Reader / Keyboard Navigation Issues**
- Hitting enter on the question mark brings up a modal with no focus and does not provide information. The modal cannot be focused and cannot be closed with keyboard navigation.
- When first focusing on the payment option, the option being selected is not described.
- The Add Honor/Memory button triggers a modal but does not focus on the modal.
- Hitting Give now brings up a new screen, but does not focus or explain what happened.
- On the Payment Info screen, tabbing past the question mark will put the focus on the Payment Type drop down arrow, but I think it should go directly to the Payment Type button itself, like with the other How Often Will You Donate and Date buttons.
- Focusing on the Payment Type button does not read "Payment Type" aloud, so you're unable to determine what you're currently focused on. It also does not read the Payment Type options when you focus on them or select them.
- Tabbing past the Payment Type button will sometimes focus on the How Often Will You Donate button, and sometimes go directly into the How Often Will You Donate search field. When the latter happens it does not say which button you're focused on. It also does not read the How Often Will You Donate options when you focus on them or select them.
- Focusing on a date in the Start Date and End Date fields doesn't describe which date you're currently on until after you've selected it.
- Tabbing past the Additional Comments field will sometimes focus on the Select your Employer... button, and sometimes go directly into the Select your Employer... search field. When the latter happens it does not say which button you're focused on.
- Focusing on Add Honor/Memory Info does not give any information on what this field is.
- When filling out Honor/Memory information, there's no indicator that you're filling out the information for the address to send the acknowledgement.
